apps/docs/content/guides/local-development/testing/overview.mdx

```diff
@@ -0,0 +1,298 @@
+
+This example demonstrates setting up and testing RLS policies for a simple todo application:
+
+1. Create a test table with RLS enabled:
+
+```sql
+-- Create a simple todos table
+create table todos (
+  id uuid primary key default gen_random_uuid(),
+  task text not null,
+  user_id uuid references auth.users not null,
+  completed boolean default false
+);
+
+-- Enable RLS
+alter table todos enable row level security;
+
+-- Create a policy
+create policy "Users can only access their own todos"
+on todos for all -- this policy applies to all operations
+to authenticated
+using (auth.uid() = user_id);
+```
+
+2. Set up your testing environment:
+
+```bash
+# Create a new test for our policies using supabase cli
+supabase test new todos_rls.test
+```
+
+3. Write your RLS tests:
+
+```sql
+begin;
+-- install tests utilities
+-- install pgtap extension for testing
+create extension if not exists pgtap with schema extensions;
+-- Start declare we'll have 6 test cases in our test suite
+select plan(6);
+
+-- Setup our testing data
+-- Set up auth.users entries
+insert into auth.users (id, email) values
+    ('123e4567-e89b-12d3-a456-426614174000', 'user1@test.com'),
+    ('987fcdeb-51a2-43d7-9012-345678901234', 'user2@test.com');
+
+-- Create test todos
+insert into public.todos (task, user_id) values
+    ('User 1 Task 1', '123e4567-e89b-12d3-a456-426614174000'),
+    ('User 1 Task 2', '123e4567-e89b-12d3-a456-426614174000'),
+    ('User 2 Task 1', '987fcdeb-51a2-43d7-9012-345678901234');
+
+-- Test 1: Verify RLS is enabled
+select results_eq(
+    $$SELECT relrowsecurity FROM pg_class WHERE relname = 'todos' AND relnamespace = 'public'::regnamespace$$,
+    ARRAY[true],
+    'Row Level Security should be enabled on todos table'
+);
+
+-- Test 2: Verify policy exists
+select policies_are('public', 'todos', ARRAY['Users can only access their own todos']);
+
+-- as User 1
+set local role authenticated;
+set local request.jwt.claim.sub = '123e4567-e89b-12d3-a456-426614174000';
+
+-- Test 3: User 1 should only see their own todos
+select results_eq(
+    'select count(*) from todos',
+    ARRAY[2::bigint],
+    'User 1 should only see their 2 todos'
+);
+
+-- Test 4: User 1 can create their own todo
+select lives_ok(
+    $$insert into todos (task, user_id) values ('New Task', '123e4567-e89b-12d3-a456-426614174000'::uuid)$$,
+    'User 1 can create their own todo'
+);
+
+-- as User 2
+set local request.jwt.claim.sub = '987fcdeb-51a2-43d7-9012-345678901234';
+
+-- Test 5: User 2 should only see their own todos
+select results_eq(
+    'select count(*) from todos',
+    ARRAY[1::bigint],
+    'User 2 should only see their 1 todo'
+);
+
+-- Test 6: User 2 cannot modify User 1's todo
+SELECT results_ne(
+    $$ update todos set task = 'Hacked!' where user_id = '123e4567-e89b-12d3-a456-426614174000'::uuid returning 1 $$,
+    $$ values(1) $$,
+    'User 2 cannot modify User 1 todos'
+);
+
+select * from finish();
+rollback;
+```
+
+4. Run the tests:
```

Comment:
```suggestion
Run the tests:
```

All these sub-numberings turn out to be 1 in the preview. So I think it's ok to remove them.